lib/server.js

```diff
@@ -238,7 +243,7 @@ function handleResponse(opts, req, resp, response) {
         }
     } else {
         opts.log('error/request', {
-            req: req,
+            root_req: req,
```

Comment:
Since this uses the child logger we've created for this request, `root_req` field will be included by default. By overwriting it here you're bypassing logged headers whitelist we've set up when we created the child logger.